location-service/src/main/java/com/telecominfraproject/wlan/location/service/LocationServiceController.java

```diff
@@ -182,6 +182,21 @@ public ListOfLocations getAllDescendants(@RequestParam long locationId) {
         }
     }
     
+    @RequestMapping(value = "/allAncestors", method = RequestMethod.GET)
+    public ListOfLocations getAllAncestors(@RequestParam long locationId) {
```

Comment:
please add this new method to the location-service-openapi.yaml

Reply:
I will do it in next update, thanks!